doc/classes/CanvasItem.xml

```diff
@@ -56,6 +56,7 @@
 			<param index="2" name="char" type="String" />
 			<param index="3" name="font_size" type="int" default="16" />
 			<param index="4" name="modulate" type="Color" default="Color(1, 1, 1, 1)" />
+			<param index="5" name="oversampling" type="float" default="0.0" />
```

Comment:
You could mention what the new argument does. I assume it's similar to the override in viewport.